packages/lib-classifier/src/store/DrawingStore.js

```diff
@@ -1,56 +1,156 @@
-import { addDisposer, getRoot, onAction, types } from 'mobx-state-tree'
+import cuid from 'cuid'
+import { addDisposer, getRoot, getType, onAction, types } from 'mobx-state-tree'
 import { autorun } from 'mobx'
 import { Subject } from 'rxjs'
 
+import { Line, Point } from './markings'
+
+const markModels = {
+  line: Line,
+  point: Point
+}
+
+function getMarkModel (toolType) {
+  return markModels[toolType] || null
+}
+
 const DrawingStore = types
   .model('DrawingStore', {
-    activeDrawingTool: types.optional(types.number, 0)
+    activeDrawingToolIndex: types.optional(types.number, 0),
```

Comment:
The active tool index seems like it should more naturally be the property of a drawing task.